jupiterone/resource_rule.go

```diff
@@ -669,6 +704,19 @@ func (r *RuleModel) BuildUpdateReferencedQuestionRuleInstanceInput() (client.Upd
 		return rule, err
 	}
 
+	if len(r.Labels) > 0 {
+		labels := make([]client.RuleInstanceLabelInput, len(r.Labels))
+
+		for i, label := range r.Labels {
+			labels[i] = client.RuleInstanceLabelInput{
+				LabelName:  label.LabelName.ValueString(),
+				LabelValue: label.LabelValue.ValueString(),
```

Comment:
How does this handle empty string?

Reply:
It'll throw a tf error 
<img width="691" alt="image" src="https://github.com/user-attachments/assets/4cb5f874-ad03-4156-95a5-69f168ed0aaa">